script/Arcueid/Day 1/01_00_ARC01_2_1.txt

```diff
@@ -21,7 +21,7 @@ The cafeteria is almost full.
 [sha:1bd9152b52c79dc4771db97348a1956f9a90c80b]{
 -- Page 137, Offset 1514.
 -- 注文待ちに10分の行列ができている。
-There's a ten-minute line waiting to order.
+You have to wait ten minutes in line to order.
```

Comment:
Maybe prefer original, it's natural to refer to lines in minute lengths in EN

Reply:
There's a ten minute queue to order.

Reply:
I remember when I first read this it literally made me do a double take.
'Does English work like this?' 'Am I crazy?'
I changed it just in case.

Reply:
English do be weird sometimes